src/library/trackset/baseplaylistfeature.cpp

```diff
@@ -336,13 +391,13 @@ void BasePlaylistFeature::slotDeletePlaylist() {
         return;
     }
 
-    if (m_lastRightClickedIndex.isValid()) {
-        VERIFY_OR_DEBUG_ASSERT(playlistId >= 0) {
-            return;
-        }
+    int nextId = selectSiblingPlaylistId(m_lastRightClickedIndex);
+
+    m_playlistDao.deletePlaylist(playlistId);
+    activate();
```

Comment:
It feels wrong to activate a unrelated sibling playlist when currently something else like AutoDj is selected.